microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -35,17 +35,50 @@ The Payment Request API⁠⁠ handles the actual payment transaction when a purc
 See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
 
 
+<!-- ====================================================================== -->
+## Enable the Digital Goods API
+
+The Digital Goods API is currently available in pre-release versions of Microsoft Edge.  To use the API, either enable the feature locally in Microsoft Edge, or register in the Digital Goods API origin trial to test the API in production.
+
+
+<!-- ------------------------------ -->
+#### Enable the feature locally
+
+To enable the Digital Goods API on your developer machine only, for testing:
+
+The `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi` flags are currently enabled as part of a running experiment, and will be enabled by default in the code once this feature reaches Edge Stable release.  You can start using the API now by running Edge Dev or Edge Canary version 115.0.1860.0 or later.
+
+Alternatively, you can use the API in your own Beta and Stable version of Microsoft Edge by launching Edge with the following command-line argument:
+
+`--enable-features=msWebAppDigitalGoods, AppStoreBilling, DigitalGoodsApi`
+
+
+<!-- ------------------------------ -->
+#### Use the origin trial to test the feature in production
+
+To test the Digital Goods API in production, register in the origin trial.  See: 
+* [Digital Goods API](https://developer.microsoft.com/microsoft-edge/origin-trials/trials/4b4a9ead-d912-4349-87b3-25e5e50b4f13) at _Microsoft Edge Developer_ > _Origin Trials_.
+* [Registering for an origin trial](../../origin-trials/index.md#registering-for-an-origin-trial) in _Use origin trials in Microsoft Edge_.
+
+To use this API, you must install a prerelease version 115.0.1860.0 of Microsoft Edge (or a later release), such as:
+* Edge Dev 115.0.1860.0
+* Edge Canary 115.0.1860.0
+
+See also:
+* [Experimental features and origin trials for PWAs](./origin-trials.md)
+
+
 <!-- ====================================================================== -->
 ## Checking whether the Digital Goods API is available
 
 To detect whether you've correctly enabled the API on your website, check for the `getDigitalGoodsService` method in the window object:
```

Comment:
Add Ref url for window object > getDigitalGoodsService method

Is the present article or article section the draft of the Ref docs for `window` object > `getDigitalGoodsService` method?
@runyuany